openpype/hosts/maya/plugins/load/load_redshift_proxy.py

```diff
@@ -137,6 +137,14 @@ def create_rs_proxy(self, name, path):
         cmds.connectAttr("{}.outMesh".format(rs_mesh),
                          "{}.inMesh".format(mesh_shape))
 
+        # TODO: use the assigned shading group as shaders if existed
+        # assign default shader to redshift proxy
+        shader_grp = next((shader_group for shader_group
+                           in cmds.ls(type="shadingEngine")
+                           if shader_group=="initialShadingGroup"))
```

Comment:
You're looking for an exact match for `initialShadingGroup`.

I think you just meant to do:
```
if cmds.objExists("initialShadingGroup"):
```

Or if you want to ensure it actually is a shading engine - which is very likely since it's a default node. (You can only delete it if you go about it the hard way via the API). But if you want to check existence and its type, just do:
```python
if cmds.ls("initialShadingGroup", type="shadingEngine"):
    ...
```